drivers/sdhc/rcar_mmc.c

```diff
@@ -657,11 +716,11 @@ static int rcar_mmc_sd_buf_rx_tx_data(const struct device *dev,
 			uint8_t copy_size = MIN(sd_buf0_size, data->block_size - w_off);
 
 			if (is_read) {
-				buf0 = sys_read64(DEVICE_MMIO_GET(dev) + RCAR_MMC_BUF0);
+				buf0 = rcar_mmc_read_buf0(dev);
```

Comment:
Just out of curiosity, why this patches are not squashed?

Reply:
at the first time they were, but I split one commit into two: one for handling unaligned access and another for working with different sd_buf0 access width sizes

if it is needed, I can combine them with one another